AnimateQuotes.js

```diff
@@ -0,0 +1,91 @@
+import React, { useRef, useEffect, useState } from 'react';
+import { Animated, Text, View } from 'react-native';
+import { useIsShake } from './useIsShake';
+import { QUOTES_IN_DURATION, QUOTES_IN_DELAY, QUOTES_OUT_DURATION, SVG_IN_DELAY, SVG_IN_DURATION, SVG_OUT_DURATION, TRANSITION_TIME_2 } from './constants';
+
+export default function AnimateQuotes(props) {
+  const [isRegistered, setIsRegistered] = useState(false);
+  const [hasToggledBefore, setHasToggledBefore] = useState(false);
+
+  const fadeAnim = useRef(new Animated.Value(0)).current;
+  const sizeAnim = useRef(new Animated.Value(0.5)).current;
+
+  const { toggle } = useIsShake();
+
+  // lands on Quotes after LogoSvg fades out first time
+  if (toggle === true && hasToggledBefore === false && isRegistered === false) {
+    console.log("AnimateQuotes, 1st if block");
```

Comment:
Typically I wouldn't want to merge in logs to a `main` branch. I understand that this can be useful for debugging, but let's keep `main` clear of logs